server/serverConstants.js

```diff
@@ -29,6 +29,8 @@ E.OPENTOK_API_KEY = { envVar: 'TB_API_KEY', jsonPath: 'OpenTok.apiKey', required
 
 E.OPENTOK_API_SECRET = { envVar: 'TB_API_SECRET', jsonPath: 'OpenTok.apiSecret', required: true };
 
+E.OPENTOK_JS_URL = { envVar: 'TB_JS_URL', jsonPath: 'OpenTok.jsUrl', required: false, defaultValue: 'https://static.opentok.com/v2/js/opentok.js' };
+
```

Comment:
Since the default is going to be static.opentok.... and we want the OpenTokDemo instance to run enterprise.opentok... we need to make sure to add the heroku conf var before merging this to Master.